generators/server/templates/pom.xml.ejs

```diff
@@ -31,9 +31,16 @@
         <!-- Build properties -->
         <exec.mainClass><%=packageName%>.<%=mainClass%></exec.mainClass>
         <maven.version>3.3.9</maven.version>
-        <java.version>1.8</java.version>
-        <node.version>v12.14.0</node.version>
-        <npm.version>6.13.7</npm.version>
+        <java.version><%= JAVA_VERSION %></java.version>
+<%_ if (!skipClient) { _%>
+        <node.version>v<%= NODE_VERSION %></node.version>
+    <%_ if (clientPackageManager === 'npm') { _%>
+        <npm.version><%= NPM_VERSION %></npm.version>
+    <%_ } _%>
+    <%_ if (clientPackageManager === 'yarn') { _%>
```

Comment:
Have we tested Yarn?

Reply:
Not yet, but I saw that the upstream generator supported it, so I pulled it in.  It uses the same Maven plugin as npm